packages/react/runtime/src/alog/render.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+import { options } from 'preact';
+import type { ComponentClass, VNode } from 'preact';
+
+import { DIFFED, DOM } from '../renderToOpcodes/constants.js';
+import type { SnapshotInstance } from '../snapshot.js';
+import { getDisplayName } from '../utils.js';
+
+export function initRenderAlog(): void {
+  const oldAfterDiff = options[DIFFED];
+  options[DIFFED] = function(vnode: VNode & { [DOM]: SnapshotInstance }) {
+    // Only log on component vnode
+    if (typeof vnode.type === 'function') {
```

Comment:
@hzy 

Using `_render` is not not correct because on each diff, a new vnode will be created by [`constructNewChildrenArray`](https://github.com/preactjs/preact/blob/df481405b9454bbbc52a235e3a90bb4363852fb7/src/diff/children.js#L152) and the `__e` will always be undefined before the render.

I choose to use the `diffed` hook and a condition check to ensure it is a component (same check as the `diff` function https://github.com/preactjs/preact/blob/df481405b9454bbbc52a235e3a90bb4363852fb7/src/diff/index.js#L81)